librz/core/cmd/cmd_eval.c

```diff
@@ -68,16 +101,87 @@ RZ_API bool rz_core_theme_load(RzCore *core, const char *name) {
 	char *system_themes = rz_path_system(RZ_THEMES);
 	char *home_file = rz_file_path_join(home_themes, name);
 	char *system_file = rz_file_path_join(system_themes, name);
+
 	free(system_themes);
 	free(home_themes);
 
-	if (!load_theme(core, home_file)) {
+	if (state->mode == RZ_OUTPUT_MODE_LONG /*&& (question_mark_count == 1)*/) {
+
+		// populate the structure RzThemeInfo only when second argument is present abcd
+
+		RzThemeInfo *MetaInfo = malloc(2 * sizeof(RzThemeInfo));
+		MetaInfo->additionaldts = NULL;
+		MetaInfo->refs = NULL;
+		char *system_file_cpy;
+		system_file_cpy = rz_str_dup(system_file_cpy, system_file);
+
+		FILE *path = rz_sys_fopen(system_file_cpy, "r");
+
+		if (!path) {
+			RZ_LOG_ERROR("Error in opening colourscheme file at '%s'\n", system_file_cpy);
+			failed = true;
+			return !failed;
+		} else {
+			rz_cons_printf("%s", "Color theme details!\n");
+		}
+
+		bool beginflag = false;
+		char line[256];
+		while ((fgets(line, sizeof(line), path) != NULL) /*(rz_file_slurp_line(path,line,0)!=NULL)*/ && (rz_str_cmp(line, "# ATEM\n", -1) != 0)) {
+
+			if (beginflag == true) {
+
+				if (strstr(line, "Author") != NULL) {
+					MetaInfo->author = strdup(line);
+				}
+
+				if (strstr(line, "License") != NULL) {
+					MetaInfo->license = strdup(line);
+				}
+
+				if (strstr(line, "Background") != NULL) {
+					MetaInfo->targetbg = strdup(line);
+				}
+
+				if (strstr(line, "depth") != NULL) {
+					MetaInfo->mincolordepth = strdup(line);
+				}
+
+				if (strstr(line, "Description") != NULL) {
+					MetaInfo->desc = strdup(line);
+				}
+
+				if (strstr(line, "Details") != NULL) {
+					MetaInfo->additionaldts = strdup(line);
+				}
+
+				if (strstr(line, "References") != NULL) {
+					MetaInfo->refs = strdup(line);
+				}
+			}
+			if (rz_str_cmp(line, "# META\n", -1) == 0) {
+				beginflag = true;
+			}
+		}
+
+		if (print_meta_info(core, state, MetaInfo, name)) {
+			metainfo_shown = true;
+		} else {
+			failed = true;
+		}
+	}
+	/* The metainfo_shown boolean was used because making the below if statement into an else statement kept throwing errors */
+
+	if ((!(load_theme(core, home_file)) || (question_mark_count > 1)) && (metainfo_shown == false)) {
+
 		if (load_theme(core, system_file)) {
 			core->curtheme = rz_str_dup(core->curtheme, name);
 		} else {
 			if (load_theme(core, name)) {
 				core->curtheme = rz_str_dup(core->curtheme, name);
-			} else {
+			}
+
```

Comment:
please pay attention to all these empty new lines
```suggestion
```

Reply:
Sorry about that, ran a proper clang-format-15 on it, which should have hopefully fixed the formatting issues